alhimick-prog/hometask-5/4.2.sql

```diff
@@ -0,0 +1 @@
+SELECT title FROM film WHERE language_id IN (1, 6, 2);
```

Comment:
@alhimick-prog `IN (1, 6, 2)` looks like chittering 😄 What if I will change ids for these languages?

Reply:
@lincAlinc i may be a cheater, but I'm not chittering.